From ae1d5a7e30ad9c48259bf56e6e2b79a9d0c126b0 Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Fri, 8 Dec 2017 14:42:05 +0200
Subject: [PATCH 3053/5242] MLK-17140-2: drm: imx: dcss: Change CTXLD trigger
 values

commit  45751d6547e896880fd800f35dbe8af864a33b0a from
https://source.codeaurora.org/external/imx/linux-imx.git

After activating the PM QoS, the old triggers didn't work anymore. Also,
this will remove a hardcoded value that might not work for all
resolutions.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dcss/dcss-dtg.c |    8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/drivers/gpu/imx/dcss/dcss-dtg.c b/drivers/gpu/imx/dcss/dcss-dtg.c
index 0291f46..9fa5614 100644
--- a/drivers/gpu/imx/dcss/dcss-dtg.c
+++ b/drivers/gpu/imx/dcss/dcss-dtg.c
@@ -249,13 +249,7 @@ void dcss_dtg_sync_set(struct dcss_soc *dcss, struct videomode *vm)
 	dtg->dis_ulc_x = dis_ulc_x;
 	dtg->dis_ulc_y = dis_ulc_y;
 
-	/*
-	 * FIXME: the context loader trigger values should not be hard-coded,
-	 * but these values proved to provide the best user experience. Planes
-	 * that use RGB pixel formats (more bandwidth) will not disappear
-	 * anymore.
-	 */
-	dcss_dtg_write(dtg, 0x0008000a, DCSS_DTG_TC_CTXLD);
+	dcss_dtg_write(dtg, dis_ulc_y, DCSS_DTG_TC_CTXLD);
 }
 EXPORT_SYMBOL(dcss_dtg_sync_set);
 
-- 
1.7.9.5

